### PR TITLE
Adrien defer fig.27 Modification & README

### PR DESCRIPTION
**Pull request by AdrienDefer**
Makefile Editing : gituhb - > github
principles/dv.rst : D=1 -> D=0

### CODE REVIEW

File: principles/dv.rst [hunk 1]
```diff
@@ -287,7 +287,7 @@ At this point, all routers have a routing table allowing them to reach all other
         \draw[dashed] (RTE) -- (E);
 
         \draw[orange, arrow] ([xshift=1.5em] D.north) -- ([xshift=1.5em] A.south) node [midway] (msg1) {};
-        \draw ([xshift=1em]msg1) -- ([xshift=1em]msg1) node [ rotate=90, rectangle, draw, font=\scriptsize] {[A=1, B=2, C=2, D=1, E=1]};
+        \draw ([xshift=1em]msg1) -- ([xshift=1em]msg1) node [ rotate=90, rectangle, draw, font=\scriptsize] {[A=1, B=2, C=2, D=0, E=1]};
```
Comment (qdeconinck) [inline principles/dv.rst:290]: You also need to update line 214 with this

**Review by qdeconinck — CHANGES_REQUESTED** [review]
You need to update the text along with the figure

File: principles/dv.rst (hunk 1 above)
Comment (AdrienDefer) [reply principles/dv.rst:290]: Done ! :)

**Comment by qdeconinck** [comment]
Fixes #37, LGTM, thanks!

**Pull request merged**